### ISSUE TITLE
The multibyte role `short_description` causes warning "Title underline too short."

### ISSUE THREAD

**Issue by akira6592** [ansible-community/antsibull-docs#228]
The multibyte role `short_description` causes warning "Title underline too short."

`argument_specs.yml`: 

```yaml
---
argument_specs:
  main:
    short_description: short description of Main entry point for テストロール
    description:
      - description of Main entry point for テストロール
    author:
      - akira6592
    options:
      myapp_int:
        type: "int"
        required: false
        default: 42
        description:
          - "The integer value, defaulting to 42."
```

```sh
% dest/build.sh

...(omit)...
/Users/akira/ansible/dest/rst/collections/local/mycollection/myapp_role.rst:19: WARNING: Title underline too short.

local.mycollection.myapp role -- short description of Main entry point for テストロール
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
/Users/akira/ansible/dest/rst/collections/local/mycollection/myapp_role.rst:41: WARNING: Title underline too short.

Entry point ``main`` -- short description of Main entry point for テストロール
------------------------------------------------------------------------

...(omit)...
```

It is not only role `short_description`, but maybe also other `short_description`.

**Comment by felixfontein** [comment]
docutils emits that warning when `column_width(title) > len(underline)`, with `from docutils.utils import column_width`. Instead of recreating `column_width`, let's simply import it from docutils (which we depend on anyway) and use it instead of `len(...)` (what we use right now).

**Comment by felixfontein** [comment]
I created #229 to fix this.

**Comment by felixfontein** [comment]
antsibull-docs 2.6.1 fixes this.

**Comment by akira6592** [comment]
@felixfontein 
Thanks for the quick fix and release!
I have confirmed that the problem has been fixed in version 2.6.1.
